README.md

```diff
@@ -27,7 +29,6 @@ To get a local copy up and running follow these simple example steps.
 ### Prerequisites
 
 ### Setup
-
 ### Install
 
 ### Usage
```

Comment:
- [OPTIONAL] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. It is a good thing you are using the template from Microverse but kindly adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your project.